LICENSE.md

```diff
@@ -1,4 +1,8 @@
-Each sub-project folder has its own license terms.
+Each project subdirectory has its own license terms.
 
-* gnovm, gno.land - copyleft derived from AGPL3.0
-* tm2 - Apache2.0
+AGPL 3.0 (copyleft)
```

Comment:
it's not AGPL, but a Gno fork, derived from AGPL

Reply:
Sounds good, I'll adjustment the language

Reply:
Would it be appropriate to state "copyleft; derivative work" or something along these lines, for cleanliness?